extensions/commands/art/cmd_remote.py

```diff
@@ -0,0 +1,184 @@
+            content = content.decode()
+
+        content_type = response.headers.get("content-type")
+
+        if content_type == "application/json":
+            # Errors from Artifactory looks like:
+            #  {"errors" : [ {"status" : 400, "message" : "Bla bla bla"}]}
+            try:
+                data = json.loads(content)["errors"][0]
+                content = "{}: {}".format(data["status"], data["message"])
+            except Exception:
+                pass
+        elif "text/html" in content_type:
+            content = "{}: {}".format(response.status_code, response.reason)
+
+        return content
+
+    except Exception:
+        return response.content
+
+
+def api_request(type, request_url, user=None, password=None, apikey=None, json_data=None):
+    headers = {}
+    if json_data:
+        headers.update({"Content-Type": "application/json"})
+    if apikey:
+        headers.update({"X-JFrog-Art-Api": apikey})
+
+    requests_method = getattr(requests, type)
+    if user and password:
+        response = requests_method(request_url, auth=(
+            user, password), data=json_data, headers=headers)
+    elif apikey:
+        response = requests_method(
+            request_url, data=json_data, headers=headers)
+    else:
+        response = requests_method(request_url)
+
+    if response.status_code == 401:
+        raise ConanException(response_to_str(response))
+    elif response.status_code not in [200, 204]:
+        raise ConanException(response_to_str(response))
+
+    return response_to_str(response)
+
+
+def read_remotes():
+    path = os.path.join(os.path.dirname(__file__), REMOTES_FILE)
+    remotes = []
+    if os.path.exists(path):
+        with open(path) as remotes_file:
+            data_encoded = remotes_file.read()
+            data = base64.b64decode(data_encoded).decode('utf-8')
+            remotes_data = json.loads(data)
+            remotes = remotes_data["remotes"]
+    return remotes
+
+
+def write_remotes(remotes):
+    path = os.path.join(os.path.dirname(__file__), REMOTES_FILE)
+    with open(path, "w") as remotes_file:
+        data = json.dumps({"remotes": remotes})
+        remotes_file.write(base64.b64encode(data.encode('utf-8')).decode('utf-8'))
+
+
+def assert_new_remote(remote_name, remotes):
+    for r in remotes:
+        if remote_name == r["name"]:
+            raise ConanException(f"Remote '{remote_name}' ({r['url']}) already exist. "
+                                 f"You can remove it using `conan art:remote remove {remote_name}`")
+
+
+def assert_existing_remote(remote_name, remotes):
+    remote_names = [r["name"] for r in remotes]
+    if remote_name not in remote_names:
+        raise ConanException(f"Remote '{remote_name}' does not exist.")
+
+
+def add_default_arguments(subparser):
+    subparser.add_argument("name", help="Name of the remote to add")
+    return subparser
+
+
+@conan_command(group="Custom commands")
+def remote(conan_api: ConanAPI, parser, *args):
```

Comment:
WDYT @czoido? In my opinion, remote command is nice because it is familiar to users but I also found asymmetry with conan remotes (repositories) in artifactory. Here is the previous conversation where we talked about the naming: https://github.com/conan-io/conan-extensions/issues/36#issuecomment-1545515516

Reply:
Yes, it's ok for me, we can call the `server` to align with the JFrog CLI